internal/pkg/cli/app_package.go

```diff
@@ -53,6 +105,36 @@ func (opts *PackageAppOpts) Execute() error {
 	return nil
 }
 
+func (opts *PackageAppOpts) listAppNames() ([]string, error) {
+	names, err := opts.ws.AppNames()
+	if err != nil {
+		return nil, fmt.Errorf("list applications in workspace: %w", err)
```

Comment:
Wasn't there previous PR where the verb tense of the errors was confusing? Did that get resolved?

Reply:
See previous comment 😝 https://github.com/aws/amazon-ecs-cli-v2/pull/250#discussion_r341383402